### DISCUSSION TITLE
Google search console sitemap couldn't featch

### DISCUSSION THREAD

**Discussion by cutbypham** [satnaing/astro-paper#185] [Q&A]
## google search console

![image](https://github.com/satnaing/astro-paper/assets/40050527/44cd0f95-a17f-46c7-9133-3ccc07fe343e)

I don't know what the problem is

## Bing webmaster

![image](https://github.com/satnaing/astro-paper/assets/40050527/110e4f46-6ff6-4346-9877-0f8cb6228777)

It worked 

---  

```js
import type { APIRoute } from "astro";
import { SITE } from "@config";

const robots = `
User-agent: *
Allow: /

Sitemap: ${new URL("sitemap-index.xml", SITE.website).href}
`.trim();

export const GET: APIRoute = () =>
  new Response(robots, {
    headers: { "Content-Type": "text/plain" },
  });
```

Website source code: https://github.com/licitfree/licitfree

I deploy on Cloudflare

link:  https://licitfree.pages.dev/

**updated**: i change my url to <https://thuantanphamfilms.pages.dev/> but the problem still here

**Comment by satnaing** [comment]
When I check your sitemap, it seems to be working fine.
Also I did check with [this](https://www.xml-sitemaps.com/validate-xml-sitemap.html).

One thing I noticed in your first screenshot is that you included `robots.txt` in your sitemap section. Maybe it is the reason. 
Besides, `robots.txt` should not be there. See [here](https://support.google.com/webmasters/thread/9688866?hl=en&msgid=9690738).
So, can you try removing `robots.txt` from search console sitemap section and retry adding sitemaps only?

**Reply by cutbypham** [reply]
![image](https://github.com/satnaing/astro-paper/assets/40050527/7209ac9c-5ea0-449f-a416-0286fa0b8a2e)
it seen like not the problem, i removed robots.txt and resubmit index sitemap again but it not worked

**Reply by satnaing** [reply]
Have you check your site url in [URL Inspection](https://support.google.com/webmasters/answer/9012289?hl=en&sjid=10070800067963255824-AP#test_live_page) tool?
Please check whether Google can fetch your url https://licitfree.pages.dev/ in URL Inspection.

**Reply by cutbypham** [reply]
Just the index page are indexed

I create new repo with astro paper template and change the site config to deploy url and deploy it on cloudflare but seen like the problem still here

![image](https://github.com/satnaing/astro-paper/assets/40050527/62002b40-c4dd-420f-8568-7ed4cd79afca)

**Reply by cutbypham** [reply]
![image](https://github.com/satnaing/astro-paper/assets/40050527/63dbbab8-c7d6-43e5-aadd-0e6b4b6b119e)

Now I deploy it on Vercel it work fine :D

**Reply by ananymishradev** [reply]
@cutbypham i have depolyed my project on vercel and i am also facing same problem of sitemap cannot be fetched can you help me 
<img width="1302" height="789" alt="screenshot-2026-01-31_17-00-48" src="https://github.com/user-attachments/assets/79c9e91a-e91f-4cc2-bbaa-829d47a5aa40" />

**Comment by eerison** [comment]
I faced the same issue, it just work, after I add `sitemap-0.xml` :/

I guess it is not taking sub sitemap :/
